website/blog/2021-02-08-64updates.md

```diff
@@ -0,0 +1,170 @@
+
+## 🎉 API parity highlights
+
+More component properties supported by the other platforms now work on Windows! Check them out.
+
+### ⚙ `Platform.version`
+The core API `Platform` has now been extended to include Windows. This means you can improve your styles and refine your Windows app experience.
+
+``` js
+import { Platform, StyleSheet } from 'react-native';
+
+const styles = StyleSheet.create({
+  container: {
+    flex: 1,    
+    ...Platform.select({
+      Windows: {
+        backgroundColor: 'blue'
+      },
+      default: {
+        // fallback for other/unspecified platforms
+        backgroundColor: 'grey'
+      }
+    })
+  }
+});
+```
+
+### ⚙ More `Text` properties
+The `Text` component properties `backgroundColor`, `border`, and `textTransform` are now available on Windows and apply to our native text controls as you'd expect.
+
+- **backgroundColor** will set the backfill/plate color behind your text.
+- **border** encompasses all the border functionalities, like borderWidth, borderColor, etc.
+- **textTransform** can force all strings lowercase, uppercase, or capitalization depending.
+
+```js
+import React from "react";
+import { Text, StyleSheet } from "react-native";
+
+const TextTest = () => {
+  return (
+    <Text style={styles.newTextProperties}>
+      some text goes here
+    </Text>
+  );
+};
+
+const styles = StyleSheet.create({
+  newTextProperties: {
+    color: "white",
+    backgroundColor: "blue",
+    borderColor: "black",
+    borderWidth: 1,
+    textTransform: "capitalize"
+  }
+});
+
+export default TextTest;
+```
+
+### ⚙ `AccessibilityInfo`
+We've begun improving the JavaScript and native control accessibility experiences by hooking up some initial behavior for these scenarios.
+
+- [**isReduceMotionEnabled**](https://reactnative.dev/docs/accessibilityinfo#isreducemotionenabled) returns true or false depending on whether the user of the device has specified they need less motion sensitivity - *full implementation*.
+- [**announceForAccessibility**](https://reactnative.dev/docs/accessibilityinfo#announceforaccessibility) tires to find an element on the current screen to raise the event from - *partial implementation*.
+- **isTouchExplorationEnabled** will return a boolean indicating whether or now the user has a screen narrator or similar running. This feature needs more testing, but is available in an alpha state for you to use - *partial implementation*.
+
+## 🎇 Expanded documentation
+We've added a lot to our documentation too, both within the Native APIs and on other important developer topics like virtual machines and picking which C language you want your React Native app to be compiled in.

```

Comment:
`C language` - I am not sure if it is a proper way to call C++ and C# to be a 'C language'. Also, the `to be compiled in` is probably better to be replaced with `to be created in`. I think we typically do not compile to a language, but rather from a language to a binary.